pgx/src/datum/mod.rs

```diff
@@ -21,6 +21,8 @@ mod into;
 mod item_pointer_data;
 mod json;
 mod numeric;
+#[cfg(not(feature = "pg10"))]
+mod range;
```

Comment:
```suggestion
#[deny(unsafe_op_in_unsafe_fn)]
mod range;
```

Reply:
should i strip all the `pg10` conditionals?

Reply:
That would be safe, yes.